webExtensionApis/study/src/index.js

```diff
@@ -448,6 +448,10 @@ this.study = class extends ExtensionAPI {
         async getInternals() {
           return studyUtils._internals;
         },
+
+        getInternalTestingOverrides: async function getInternalTestingOverrides() {
```

Comment:
The formatting of this seems inconsistent. Why not `async getInternalTestingOverrides() {`?

Reply:
Originally all function declarations were generated based on the schema YAML in a consistent manner. I believe prettier causes the observed inconsistencies based on the length of the function names.